externs/browser/w3c_geolocation.js

```diff
@@ -27,8 +27,18 @@
 function Geolocation() {}
 
 /**
- * @param {function(!GeolocationPosition)} successCallback
- * @param {(function(!GeolocationPositionError)|null)=} opt_errorCallback
+ * @typedef {function(!GeolocationPosition): void}
+ */
+var PositionCallback;
+
+/**
+ * @typedef {function(!GeolocationPositionError): void}
+ */
+var PositionErrorCallback;
+
+/**
+ * @param {PositionCallback} successCallback
+ * @param {PositionErrorCallback} opt_errorCallback
```

Comment:
{PositionErrorCallback=}? It should be optional (in the spec, and indicated by the opt_ name)
Same for the one below in function watchPosition.

Reply:
You are completely right. It seems it got lost as I was copying this code back from our integration branch to this PR.

So sorry for wasting your time.

Force pushed a hopefully correct patch